[16.0][ADD] stock_lot_remove: Remove quants with a past removal date (#2021)
OCA/stock-logistics-workflow: stock_lot_auto_remove/readme/CONFIGURE.md

> To configure this module, you need to to enable the automatic removal of quants with a past removal date. This can be done by following these steps:

Reviewer: ```suggestion
To configure this module, you need to to enable the removal of expired lots. This can be done by following these steps:
```
